src/taipy/_cli/_scaffold_cli.py

```diff
@@ -40,5 +38,6 @@ def parse_arguments(cls):
         args = _CLI._parse()
 
         if getattr(args, "which", None) == "create":
-            cookiecutter(cls._TEMPLATE_MAP[args.template])
+            application_path = cookiecutter(cls._TEMPLATE_MAP[args.template])
+            print(f"New Taipy application is created at {application_path}")
```

Comment:
"is" -> "has been"

May be a line on what to do next would be even better (like run the app).